app/matplotlib.ipynb

```diff
@@ -86,23 +102,39 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
-   "outputs": [],
    "source": [
     "x = np.random.rand(50)\n",
     "y = np.random.rand(50)\n",
     "colors = np.random.rand(50)\n",
     "sizes = 100 * np.random.rand(50)\n",
     "\n",
-    "# write your code here"
+    "scatter = plt.scatter(x, y, c=colors, s=sizes)\n",
+    "plt.title(\"Random Scatter Plot\")\n",
+    "plt.xlabel(\"X Axis\")\n",
+    "plt.ylabel(\"Y Axis\")\n",
+    "\n",
+    "plt.show()"
```

Comment:
To complete the scatter plot task, you need to add a colorbar to the plot. You can do this by using `plt.colorbar(scatter)` after the scatter plot is created. This will help in setting the colorbar label as required by the task.